acra-censor/handlers/handlers_util.go

```diff
@@ -171,46 +171,6 @@ func handleSelectPattern(queryNodes, patternNodes []sqlparser.SQLNode) bool {
 	return false
 }
 
-// isColumnPattern return true if this SelectExpr is our %%COLUMN%% pattern
```

Comment:
why are these three functions moved?

Reply:
It's just only move without any logic change. For better readability. Now it will be:
// pattern handlers
- handleSelectPattern
- handleSelectColumnPattern
- handleSelectWherePattern
- handleStarPattern
- handleWherePatterns

// helper methods
- isColumnPattern
- isValueReplacer
- isValuePattern
- isListOfValuesPattern
- IsEqualComparisonNodes

// matchers
- matchValuePattern
- matchSubqueryPattern
- matchQueryNodeWithValuePattern
- matchRangeCondition
- matchOrderBy

And then other utilility functions

Reply:
got it!